test/util.js

```diff
@@ -380,7 +380,7 @@ describe('utility methods', function() {
 			function resolveMissingFormatter() {
 				util.resolveFormatter('missing-formatter');
 			}
-			resolveMissingFormatter.should.throw(TypeError, {message: 'Invalid Formatter'});
+			resolveMissingFormatter.should.throw(/There was a problem loading formatter/);
```

Comment:
Submitted in https://github.com/adametry/gulp-eslint/commit/fc9200e986faa2282145c8182c1aa142961bb7ec as your commit. Thanks.